testbed/scripts/benchmarks/rfcBBL102.sh

```diff
@@ -20,7 +20,7 @@ PARALLEL_GEN=100
 TESTCASE=sparse
 INPUT_DATA=random
 # DATA_DIR=../extra/testDataset
-DATA_DIR=/home/adlrocha/Desktop/main/work/ProtocolLabs/repos/beyond-bitswap/datasets/testDataset
```

Comment:
@adlrocha can you make sure that there is nothing that is tied to your own home setup. All of the testing should be "git clone and go"

Reply:
My fault! This should be fixed by now! I tried to clean every dependence from my local environment but I may have left some things behind.

Reply:
Do `ag adlrocha` to find them all ([ag](https://geoff.greer.fm/ag/)). Most are gomod pointing to forks, but there are still a couple on scripts and jupyter notebooks.

```
testbed/go.mod:1:module github.com/adlrocha/beyond-bitswap/testbed
testbed/go.mod:59:// replace github.com/ipfs/go-bitswap => /home/adlrocha/Desktop/main/work/ProtocolLabs/repos/ipfs/go-bitswap
testbed/go.mod:62:replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap v0.2.20-0.20201006081544-fad1a007cf9b
testbed/go.mod:65:// replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap v0.2.20-0.20200928082451-4186ba0c569c
testbed/go.mod:68:// replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap v0.2.20-0.20201104081555-6f5c6dc5e81b
testbed/go.mod:71:// replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap 39aeae7a7b188abfc659d49c12e27059e1e7b8ee
testbed/go.mod:74:// replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap 5f82e1f53671cdd88bb809a8a86087069d90fc08
testbed/go.mod:78:// replace github.com/libp2p/go-libp2p-core => github.com/adlrocha/go-libp2p-core bf5d45ca7e53c6c20d8f319f0f23c2fef93bf7f6
testbed/go.mod:80:// replace github.com/libp2p/go-libp2p-core => github.com/adlrocha/go-libp2p-core v0.6.2-0.20201008055055-b309947fc237
testbed/go.mod:82:// replace github.com/libp2p/go-libp2p => github.com/adlrocha/go-libp2p v0.11.1-0.20201008063238-d1909d9adebc
testbed/go.mod:84:// replace github.com/libp2p/go-libp2p-gzip => github.com/adlrocha/go-libp2p-gzip v0.0.0-20201008055912-3c159e9de091
testbed/go.mod:86:// replace github.com/libp2p/go-conn-compression-multistream => github.com/adlrocha/go-conn-compression-multistream v0.0.0-20201008054958-988bc118b907
testbed/go.mod:88:// replace github.com/libp2p/go-libp2p-transport-upgrader => github.com/adlrocha/go-libp2p-transport-upgrader v0.3.1-0.20201007105011-1e846954f580
testbed/test/graphsyncTransfer.go:12:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/test/tcpTransfer.go:13:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/test/ipfsTransfer.go:17:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/test/sparse.go:20:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/test/transfer.go:20:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/test/common.go:18:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/compositions/rfcBBL104.toml:3:author  = "adlrocha"
testbed/test/waves.go:17:	"github.com/adlrocha/beyond-bitswap/testbed/utils"
testbed/README.md:28:$ git clone https://github.com/adlrocha/beyond-bitswap.git
testbed/README.md:128:Oct  9 10:58:05.774013  INFO    using home directory: /home/adlrocha/testground
testbed/README.md:129:Oct  9 10:58:05.774168  INFO    .env.toml loaded from: /home/adlrocha/testground/.env.toml
testbed/go.sum:39:github.com/adlrocha/beyond-bitswap v0.0.0-20200820074433-a09463cff1eb h1:zznbHf5QNjk+jpaff9P9ZyxChqxM1I60tkXSiBXS3N4=
testbed/go.sum:40:github.com/adlrocha/go-bitswap v0.2.19 h1:pT+HZwyX83HAMo3+dSU+Lvh5EAOGn+u2BpJz9YP40iM=
testbed/go.sum:41:github.com/adlrocha/go-bitswap v0.2.19/go.mod h1:C7TwBgHnu89Q8sHsTJP7IhUqF9XYLe71P4tT5adgmYo=
testbed/go.sum:42:github.com/adlrocha/go-bitswap v0.2.20-0.20200806120326-85180af7f751 h1:nSdKkywMLi6oMD4zMDlygEOXCWXoCdqB5MPFLKHfieI=
testbed/go.sum:43:github.com/adlrocha/go-bitswap v0.2.20-0.20200806120326-85180af7f751/go.mod h1:C7TwBgHnu89Q8sHsTJP7IhUqF9XYLe71P4tT5adgmYo=
testbed/go.sum:44:github.com/adlrocha/go-bitswap v0.2.20-0.20200807084541-c90cd10de144 h1:yCY6FvVHPFyx4sYIBTzKsiOK3ADy4WdVWZ9Ql1IlsV4=
testbed/go.sum:45:github.com/adlrocha/go-bitswap v0.2.20-0.20200807084541-c90cd10de144/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:46:github.com/adlrocha/go-bitswap v0.2.20-0.20200811113214-8663be04154d h1:Yw/1mNkUD5XGkIMuowi8MNEaWkXBEaDJt7QlKjmyvBg=
testbed/go.sum:47:github.com/adlrocha/go-bitswap v0.2.20-0.20200811113214-8663be04154d/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:48:github.com/adlrocha/go-bitswap v0.2.20-0.20200820095049-9ae33c6eabec h1:9aD6IkJjfAsnd2If+Eg7nFq9ooOrsfGab17RtgjkmRg=
testbed/go.sum:49:github.com/adlrocha/go-bitswap v0.2.20-0.20200820095049-9ae33c6eabec/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:50:github.com/adlrocha/go-bitswap v0.2.20-0.20200901063521-a5520bd05e60 h1:3orUtDd0x2g4eTuIPJv2XRFw7p4RLiUgKmoix1bL6To=
testbed/go.sum:51:github.com/adlrocha/go-bitswap v0.2.20-0.20200901063521-a5520bd05e60/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:52:github.com/adlrocha/go-bitswap v0.2.20-0.20200909065718-674118d87e39 h1:3dhEyUypv6RhBt5+ebTdYiNTz72pA+TyBgCYBpjbyb8=
testbed/go.sum:53:github.com/adlrocha/go-bitswap v0.2.20-0.20200909065718-674118d87e39/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:54:github.com/adlrocha/go-bitswap v0.2.20-0.20200910120545-d151875a9404 h1:RRGHF/MzuLu7xLZaiMQTIzLSMVmuHmerOg8gzK8pUsw=
testbed/go.sum:55:github.com/adlrocha/go-bitswap v0.2.20-0.20200910120545-d151875a9404/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:56:github.com/adlrocha/go-bitswap v0.2.20-0.20200928082451-4186ba0c569c h1:0v9lKqtZLP/UjDEOjoPnmAZEZKGZ8TfmrBA2WlkJG60=
testbed/go.sum:57:github.com/adlrocha/go-bitswap v0.2.20-0.20200928082451-4186ba0c569c/go.mod h1:3cPec3q5Km1wfn76GaZQ8bceqeH5qUGxXDjziiNDP3Q=
testbed/go.sum:58:github.com/adlrocha/go-bitswap v0.2.20-0.20200928095440-64951107fd58 h1:ZZuFKDue+Cxci02UHkJqVWwGqXS2XR7p+1nuffVnIJQ=
testbed/go.sum:59:github.com/adlrocha/go-bitswap v0.2.20-0.20200928095440-64951107fd58/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:60:github.com/adlrocha/go-bitswap v0.2.20-0.20201002103752-9be5ba579788 h1:VP6oAmV/T5JCDRq5x0T5XNXTAtV51ACL6rWBkBfsnuA=
testbed/go.sum:61:github.com/adlrocha/go-bitswap v0.2.20-0.20201002103752-9be5ba579788/go.mod h1:1Svm9h8Tpy17eZPM0Rf9eq8WoNF6/xO9txzoUTzhFt4=
testbed/go.sum:62:github.com/adlrocha/go-bitswap v0.2.20-0.20201006081544-fad1a007cf9b h1:LsawLVDWRTTP+RqMn9e2JYGWTK9Ki2MhtPL07aQg4ns=
testbed/go.sum:63:github.com/adlrocha/go-bitswap v0.2.20-0.20201006081544-fad1a007cf9b/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=
testbed/go.sum:64:github.com/adlrocha/go-bitswap v0.2.20-0.20201020072614-345472ed2050 h1:3fkIUsfqOVilaJLNuOiG1ROQqh0b9+vDJZwcgHRx6gs=
testbed/go.sum:65:github.com/adlrocha/go-bitswap v0.2.20-0.20201020072614-345472ed2050/go.mod h1:GKc2qL+ziEK5p17Glbx6vxcyBTIAiGtGKi1fpucJqp4=
testbed/go.sum:66:github.com/adlrocha/go-bitswap v0.2.20-0.20201104081555-6f5c6dc5e81b h1:Dx45+6eo9ARBYKc/OYWpn05prhy785YnSEpejrg94gc=
testbed/go.sum:67:github.com/adlrocha/go-bitswap v0.2.20-0.20201104081555-6f5c6dc5e81b/go.mod h1:eHCUwlYYvOiVEp+mn1lyt15fb3YORqSDYGisRoLy6cA=
testbed/go.sum:68:github.com/adlrocha/go-conn-compression-multistream v0.0.0-20201008054958-988bc118b907 h1:THlrravAXb+nDWFjTS0y+rUK+OKcvdQRgrG5x7n0DbQ=
testbed/go.sum:69:github.com/adlrocha/go-conn-compression-multistream v0.0.0-20201008054958-988bc118b907/go.mod h1:0F55yqtkfblbm4keDI6cwfpW3QbPMEFDgWRPnh7mZoQ=
testbed/go.sum:70:github.com/adlrocha/go-libp2p v0.11.1-0.20201008063238-d1909d9adebc h1:LqYYRj+abbPkZRRBo0QeoCM9Oa/U4pedWDzqSEfTFR0=
testbed/go.sum:71:github.com/adlrocha/go-libp2p v0.11.1-0.20201008063238-d1909d9adebc/go.mod h1:MOzduE9LDbSKvMc+Ug3CdAqK3XUQFHwwjOeacaUAkeQ=
testbed/go.sum:72:github.com/adlrocha/go-libp2p-core v0.6.2-0.20201008055055-b309947fc237 h1:pnwlPPNI1LmXgCpdOX8wpUU3i1qOVRAZmTAUYMkqRaA=
testbed/go.sum:73:github.com/adlrocha/go-libp2p-core v0.6.2-0.20201008055055-b309947fc237/go.mod h1:FfewUH/YpvWbEB+ZY9AQRQ4TAD8sJBt/G1rVvhz5XT8=
testbed/go.sum:74:github.com/adlrocha/go-libp2p-gzip v0.0.0-20201008055912-3c159e9de091 h1:SRAKpJk0aBUe4iOv/bLAas0diC8FESLpLQUA8//bafI=
testbed/go.sum:75:github.com/adlrocha/go-libp2p-gzip v0.0.0-20201008055912-3c159e9de091/go.mod h1:Wd/b/1nwaXHs+rq4V/d0NDwV9/O40++fbjlvaoFxnmY=
testbed/go.sum:76:github.com/adlrocha/go-libp2p-transport-upgrader v0.3.1-0.20201007105011-1e846954f580 h1:2Rzhpj8jYjRwrSQXiR1WgC7w5UtfiIvyxf2ks2rROEw=
testbed/go.sum:77:github.com/adlrocha/go-libp2p-transport-upgrader v0.3.1-0.20201007105011-1e846954f580/go.mod h1:fH1YRmRiMYVyOE6YbLXQPnyWQrBq11zIF39iHqJSwIk=
testbed/scripts/benchmarks/rfcBBL104.sh:25:# DATA_DIR=/home/adlrocha/Desktop/main/work/ProtocolLabs/repos/beyond-bitswap/testbed/scripts/saved/test
testbed/scripts/benchmarks/template.sh:28:DATA_DIR=/home/adlrocha/Desktop/main/work/ProtocolLabs/repos/beyond-bitswap/testbed/scripts/saved/test
testbed/scripts/dashboard.ipynb:143:      "Oct 26 09:13:43.861285\t\u001b[34mINFO\u001b[0m\tusing home directory: /home/adlrocha/testground\n",
testbed/scripts/dashboard.ipynb:144:      "Oct 26 09:13:43.861405\t\u001b[34mINFO\u001b[0m\t.env.toml loaded from: /home/adlrocha/testground/.env.toml\n",
testbed/main.go:4:	test "github.com/adlrocha/beyond-bitswap/testbed/test"
probe/go.mod:1:module github.com/adlrocha/beyond-bitswap/probe
probe/go.mod:57:replace github.com/ipfs/go-bitswap => github.com/adlrocha/go-bitswap v0.2.20-0.20200811113214-8663be04154d
probe/go.mod:59:// replace github.com/ipfs/go-bitswap => /home/adlrocha/Desktop/main/work/ProtocolLabs/repos/go-bitswap
probe/go.sum:38:github.com/adlrocha/go-bitswap v0.2.20-0.20200811113214-8663be04154d h1:Yw/1mNkUD5XGkIMuowi8MNEaWkXBEaDJt7QlKjmyvBg=
probe/go.sum:39:github.com/adlrocha/go-bitswap v0.2.20-0.20200811113214-8663be04154d/go.mod h1:Wxv8HCi8VSTvaeMmuvWf8dg7Q2UPwMC1r3av3gCqiEk=

```